docs/domains/Domain.json

```diff
@@ -484,6 +487,41 @@
       "type": "object",
       "properties": {}
     },
+    "Model": {
+      "type": "object",
+      "properties": {
+        "encryptionSecret": {
+          "description": "WDT encryption key passphrase secret.",
+          "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.SecretReference"
+        },
+        "domainType": {
+          "description": "WDT domain type: Legal values: WLS, RestrictedJRF, JRF",
+          "type": "string",
+          "enum": [
+            "WLS",
+            "RestrictedJRF",
+            "JRF"
+          ]
+        },
+        "configMapName": {
+          "description": "WDT config map name",
+          "type": "string"
+        }
+      }
+    },
+    "Opss": {
+      "type": "object",
+      "properties": {
+        "keyWalletConfigMap": {
+          "description": "The name of the config map containing the OPSS key wallet file",
+          "type": "string"
+        },
+        "walletSecret": {
+          "description": "Secret containing OPSS key passphrase.",
+          "$ref": "https://github.com/garethr/kubernetes-json-schema/blob/master/v1.13.5/_definitions.json#/definitions/io.k8s.api.core.v1.SecretReference"
```

Comment:
I think we agreed to change this to walletFileSecret and walletPasswordSecret per slack thread https://proddev-paas-fmw.slack.com/archives/CKFA73SNM/p1580504991015300